lib/scarpe/document_root.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
+require "cgi"
+
 class Scarpe
   class DocumentRoot < Scarpe::Widget
+    include Scarpe::Background
```

Comment:
I was working on this too for getting some of the other examples working. Is there a reason we don't just include `Background` in `Widget`? Not to spec?

Reply:
I think the only widgets that are supposed to support background are `flow` and `stack`. In original shoes, the "root app block" was also a stack. But because we handle that slightly different, the `DocumentRoot` should behave as much as possible like a `stack`. That's why it's included here, but not anywhere else.